codec_specs.md

```diff
@@ -817,6 +817,16 @@ The `BlockAddId` of the `BlockMore` containing these data **MUST** be 1.
 
 Initialization: none
 
+### A_ATRAC/AT1
+
+Codec ID: A_ATRAC/AT1
+
+Codec Name: Sony ATRAC1 Codec
+
+Description: The original ATRAC codec by Sony, mainly used in MiniDisc platforms.
```

Comment:
May you add a link to a a page describing a bit the format? Even if not an official spec, at least it provides developers a starting point.

Reply:
I added a link to the original ATRAC1 paper, and a reference encoder.